Tasks/NuGetCommand/task.json

```diff
@@ -0,0 +1,447 @@
+            "defaultValue": "$(Build.ArtifactStagingDirectory)/*.nupkg",
+            "helpMarkDown": "The pattern to match or path to nupkg files to be uploaded. Multiple patterns can be separated by a semicolon.",
+            "required": true,
+            "visibleRule": "command = push"
+        },
+        {
+            "name": "nuGetFeedType",
+            "type": "radio",
+            "label": "Target feed location",
+            "required": true,
+            "defaultValue": "internal",
+            "options": {
+                "internal": "This account/collection",
+                "external": "External NuGet server (including other accounts/collections)"
+            },
+            "visibleRule": "command = push"
+        },
+        {
+            "name": "feedPublish",
+            "type": "pickList",
+            "label": "Target feed",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select a feed hosted in this account. You must have Package Management installed and licensed to select a feed here.",
+            "visibleRule": "command = push && nuGetFeedType = internal"
+        },
+        {
+            "name": "allowPackageConflicts",
+            "type": "boolean",
+            "label": "Allow duplicates to be skipped",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "If you continually publish a set of packages and only change the version number of the subset of packages that changed, use this option. It allows the task to report success even if some of your packages are rejected with 409 Conflict errors.\n\nThis option is currently only available on Visual Studio Team Services. If NuGet.exe encounters a conflict, the task will fail.",
+            "visibleRule": "command = push && nuGetFeedType = internal"
+        },
+        {
+            "name": "externalEndpoint",
+            "type": "connectedService:ExternalNuGetFeed",
+            "label": "NuGet server",
+            "required": true,
+            "helpMarkDown": "The NuGet service endpoint that contains the external NuGet server’s credentials.",
+            "visibleRule": "command = push && nuGetFeedType = external"
+        },
+        {
+            "name": "verbosityPush",
+            "type": "pickList",
+            "label": "Verbosity",
+            "defaultValue": "Detailed",
+            "helpMarkDown": "Specifies the amount of detail displayed in the output.",
+            "required": "false",
+            "groupName": "pushAdvanced",
+            "options": {
+                "-": "-",
+                "Quiet": "Quiet",
+                "Normal": "Normal",
+                "Detailed": "Detailed"
+            }
+        },
+
+        {
+            "name": "searchPatternPack",
+            "type": "filePath",
+            "label": "Path to csproj or nuspec file(s) to pack",
+            "defaultValue": "**\\*.csproj",
```

Comment:
The direction of the slash in the default value is the opposite of the other defaults